01-messaging.md

```diff
@@ -259,8 +259,12 @@ The `features` field MUST be padded to bytes with 0s.
     2. data:
         * [`...*chain_hash`:`chains`]
 
+    1. type: 2 (`remote_addr`)
```

Comment:
If you're not adding a feature bit (and I don't believe we need to add one), you must use an odd number for compatibility (such as `3` for example)